config/api.yml

```
@@ -986,18 +986,28 @@
     :options:
     - :collection
     - :subcollection
-    :verbs: *g
+    :verbs:
+    - :get
+    - :put
+    - :post
+    - :patch
```

Minor but you should be able to use


```suggestion
    :verbs: *gppp
```

---

Do we have that?  I don't see it in the list of sets

---

Seems not, but then I'd prefer adding that one new set to keep the verbs section consistent.

---

Done